pytest_fixtures/authsource_fixtures.py

```diff
@@ -0,0 +1,207 @@
+"""Test class for Active Directory Feature
```

Comment:
I don't think this metadata belongs in the fixture module, as this module is for defining related fixtures, to be used by various test modules.

Fixtures within this module may be used by test modules that are only reliant on LDAP as a prerequisite.